### PR TITLE
feat: add auto deployment FE app on cloudFront

### PR DESCRIPTION
**Pull request by Shutya**
Manually created resources:

S3 Bucket URL (403 Status): http://aws-cource-web.s3-website-eu-west-1.amazonaws.com
CloudFront URL: d1bf9ck7i9rqoe.cloudfront.net

Automated created resources:

S3 Bucket URL (403 Status): http://aws-cource-web-bucket.s3-website-us-east-1.amazonaws.com/
CloudFront URL: https://dkqjmtkbng0bb.cloudfront.net

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,34 +5,49 @@
...
+    "enzyme-to-json": "^3.4.4",
+    "formik": "^2.1.5",
+    "formik-material-ui": "^2.0.1",
+    "lodash": "^4.17.19",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-redux": "^7.2.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.1",
+    "serverless-finch": "^2.6.0",
```
Comment (generalov) [inline package.json:48]: "serverless" package should be also included into the dependencies.
take a look for example https://github.com/boale/serverlessTestApp/blob/master/package.json

File: package.json [hunk 2]
```diff
@@ -5,34 +5,49 @@
...
+    "serverless-single-page-app-plugin": "file:./serverless-single-page-app-plugin",
```
Comment (generalov) [inline package.json]: Just a tip. If you put the plugin in the `.serverless_plugins` folder, you wouldn't have to modify package.json and reinstall dependencies. https://www.serverless.com/framework/docs/providers/google/guide/plugins#service-local-plugin

**Review by generalov — COMMENTED** [review]
I cannot rate this PR yet. Please include into the PR description all required links:

Manually created resources:
* S3 Bucket URL (403 Status):
* CloudFront URL: 

Automated created resources:
* S3 Bucket URL (403 Status):
* CloudFront URL:

File: package.json (hunk 2 above)
Comment (ilya-moskovtsev) [reply package.json]: That's an interesting point. I should try that.

**Review by generalov — APPROVED** [review]

**Review by ilya-moskovtsev — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Shutya) [reply package.json:48]: Done. Thanks

File: package.json (hunk 2 above)
Comment (Shutya) [reply package.json]: Awesome, done, many thanks. It looks much more beautiful.

**Comment by Shutya** [comment]
> I cannot rate this PR yet. Please include into the PR description all required links:
> 
> Manually created resources:
> 
> * S3 Bucket URL (403 Status):
> * CloudFront URL:
> 
> Automated created resources:
> 
> * S3 Bucket URL (403 Status):
> * CloudFront URL:

Done, thanks. I'm lucky that I didn't remove manually created s3 and cloudfront.

**Pull request merged**